addons/mail/tests/test_mail_activity.py

```diff
@@ -18,3 +25,170 @@ def test_chatter_activity_tour(self):
             "mail_activity_schedule_from_chatter",
             login="admin",
         )
+
+
+@tests.tagged('mail_activity')
+class TestMailActivity(MailCommon):
```

Comment:
Better move in 'test_mail' and use test models available in that module, so that we don't depend on specific business implementation (I think I asked the same thing on the other activity task :D :disguised_face: )